src/app/api/main.py

```
@@ -116,12 +130,127 @@ async def lifespan(app: FastAPI):
+    return JobOut(
+        job_id=job.job_id,
+        trigger_type=job.trigger.type,
+        trigger_repr=str(trigger),
+    )
+
+
+
+@app.delete("/jobs/{job_id}", status_code=204)
+async def delete_job(
+    job_id: str,
+    session: Session = Depends(get_sqlmodel_session),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)